City-of-Helsinki/drupal-helfi-kasvatus-koulutus
# UHF-5298: Vocational school filters

# Filter vocational search by study field [UHF-5298](https://helsinkisolutionoffice.atlassian.net/browse/UHF-5298)
Adds study field filter for vocational school search.

## What was done
* Adds configs to create a taxonomy for vocational studies.
* Adds configs to create a taxonomy reference field for TPR unit.
* Creates initial data for vocational studies taxonomy.
* Uses the new taxonomy to filter vocational schools.

## How to install
* Make sure your instance is up and running.
* Get these changes:
    * `git pull origin UHF-5298-vocational-school-filters`
* Run `make shell` and:
  * `drush updb`
  * `drush cim`
  * `drush cr`

## How to test
* Add a page containing a vocational school search paragraph.
* Manually edit some vocational schools and add one or many study fields.
* Check that the vocational schools are filtered by the study field.
